src/mmrelay/matrix/prefixes.py

```
@@ -0,0 +1,600 @@
+import re
+from typing import Any, Dict
```

Ruff (0.15.9)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)